apps/aedevmode/src/aedevmode_app.erl

```diff
@@ -0,0 +1,28 @@
+-module(aedevmode_app).
+-behavior(application).
+
+-export([ start/2
+        , stop/1 ]).
+
+-export([post_check/0]).
+
+start(_StartType, _StartArgs) ->
+    aedevmode_sup:start_link().
+
+stop(_State) ->
+    lager:info("Stopping aedevmode_app", []),
+    ok.
+
+%% Run at 300, i.e. after normal env checks AND the db start
+post_check() ->
+    case aecore_env:is_dev_mode() of
+        true ->
+            Pub = patron_pubkey_for_testing(),
+            aeu_env:suggest_config([<<"mining">>, <<"beneficiary">>], Pub);
```

Comment:
can't see from here where this is ran, but it would be nice if it wouldn't override the mining beneficiary I set from the pool of generated (and used) accounts.

Reply:
The `post_check()` is run in start phase `300`, that is, after all the 'official' start phase hooks.
Also, the `suggest_config/2` function only sets the value _if_ it was previously undefined. So it explicitly won't override anything set previously.